etna/whatson/models.py

```diff
@@ -589,3 +630,30 @@ def save(self, *args, **kwargs):
         "whatson.WhatsOnPage",
     ]
     subpage_types = []
+
+
+class EventFilterForm(forms.Form):
+    date = forms.DateField(
+        label="Choose a date",
+        required=False,
+        widget=forms.DateInput(attrs={"type": "date", "class": "filters__date"}),
+    )
+
+    category = forms.ModelChoiceField(
+        widget=forms.RadioSelect(attrs={"class": "filters__radio"}),
+        queryset=EventType.objects.all(),
+        required=False,
+        label="What",
+    )
+
+    online = forms.BooleanField(
```

Comment:
Suggestion: `online` is quite ambiguous - would something like `is_online_event` make this more readable?